New topic: metadata (#110)
ubvu/open-handbook: topics/data-documentation.qmd

# Data Documentation
By creating documentation about your research data you can make it easier for yourself or for others to manage, find, assess and use your data. The process of documenting means to describe your data and the methods by which they were collected, processed and analysed. The documentation or descriptions are also referred to as metadata, i.e. data about data. These metadata can take various forms and can describe data on different levels.

An example that is frequently used to illustrate the importance of metadata is the use of the label on a can of soup. The label tells you what kind of soup the can contains, what ingredients are used, who made it, when it expires and how you should prepare the soup for consumption.

When you are documenting data, you should take into account that there are different kinds of metadata and that these metadata are governed by various standards. These include, but are not limited to:

> 1.	FAIR data principles: a set of principles to make data Findable, Accessible, Interoperable and Reusable. 
> 2.	Guidelines for unstructured metadata: mostly research domain-specific guidelines on how to create READMEs or Codebooks to describe data.
> 3.	Standards for structured metadata: generic or research domain-specific standards to describe data.

Reviewer: ```suggestion
1. FAIR data principles: a set of principles to make data Findable, Accessible, Interoperable and Reusable. 
2. Guidelines for unstructured metadata: mostly research domain-specific guidelines on how to create READMEs or Codebooks to describe data.
3. Standards for structured metadata: generic or research domain-specific standards to describe data.
```

![](../public/data_documentation.png)

## FAIR data principles
The FAIR data principles provide guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets. The principles emphasise machine-actionability, i.e., the capacity of computational systems to find, access, interoperate, and reuse data with none or minimal human intervention. 

> More information can be found in the section about the [FAIR data principles](https://ubvu.github.io/open-handbook/topics/fair-principles.html).

Reviewer: ```suggestion
More information can be found in the section about the [FAIR data principles](./fair-principles.qmd).
```

Making this a relative link 😊

## Unstructured metadata
Unstructured metadata are mainly intended to provide more detailed information about the data and is primarily readable for humans. The type of research and the nature of the data influence what kind of unstructured metadata is necessary. Unstructured metadata are attached to the data in a file. The format of the file is chosen by the researcher. 

### README
A README file provides information about data and is intended to ensure that data can be correctly interpreted, by yourself or by others. A README file must be submitted along with the data.

> Example of READMEs
> - [Guidelines for creating a README file – 4TU.ResearchData](https://data.4tu.nl/s/documents/Guidelines_for_creating_a_README_file.pdf)
> - [Guide to writing "readme"-style metatada - Cornell Data Services](https://data.research.cornell.edu/data-management/sharing/readme/)

Reviewer: ```suggestion
Example of READMEs

- [Guidelines for creating a README file – 4TU.ResearchData](https://data.4tu.nl/s/documents/Guidelines_for_creating_a_README_file.pdf)
- [Guide to writing "readme"-style metatada - Cornell Data Services](https://data.research.cornell.edu/data-management/sharing/readme/)
```

As far as I know the list is fine with either `-` or `*` in Markdown (although I did not check with respect to `.qmd` files). Important is to have a whiteline before and after.

Author: Thank you, Chris! I've used `-` in Markdown before, indeed, and I was surprised it didn't work.

### Codebook
A Codebook is another way to describe the contents, structure and layout of the data. A well documented codebook is intended to be complete and self-explanatory and contains information about each variable in a data file. A codebook must be submitted along with the data.

Examples of codebooks are
- [Creating a codebook - Kent State University](https://libguides.library.kent.edu/SPSS/Codebooks)
- [Codebook - Amsterdam Public Health](https://aph-qualityhandbook.org/set-up-conduct/methods-data-collection/2-2-quantitative-research/2-2-1-preparation/codebook/)
- [DDI-Codebook - Data Documentation Initiative Alliance](https://ddialliance.org/Specification/DDI-Codebook/) 

## Structured metadata
Structured metadata are intended to provide information in a standardised way. The structured metadata are readable for both humans and machines. It can be used by data catalogues, for example DataCite Commons. 

The standardisation of metadata involves the following aspects:

> 1.	Elements: rules about the fields that must be used to describe an object, for example the <title>, <author> and <publicationDate>. 
> 2.	Values: rules about the values that must be used within specific elements. Controlled vocabularies, classifications and Persistent Identifiers are used to reduce ambiguity and ensure consistency, for example by using a term from a controlled vocabulary like the Medical Subject HEadings (MeSH) as a <subject> and an Perstent Identifier such as an ORCID to identify a <person>.
> 3.	Formats: rules about the formats used to exchange metadata, for example JSON or XML.

Reviewer: ```suggestion
1. Elements: rules about the fields that must be used to describe an object, for example the <title>, <author> and <publicationDate>. 
2. Values: rules about the values that must be used within specific elements. Controlled vocabularies, classifications and Persistent Identifiers are used to reduce ambiguity and ensure consistency, for example by using a term from a controlled vocabulary like the Medical Subject HEadings (MeSH) as a <subject> and an Perstent Identifier such as an ORCID to identify a <person>.
3. Formats: rules about the formats used to exchange metadata, for example JSON or XML.
```

### Metadata standards

> Metadata standards allow for easier exchange of metadata and harvesting of the metadata by search engines. Many certified archives use a metadata standard for the descriptions. If you choose a data repository or registry, you should find out which metadata standard they use. At the VU the following standards are used:
> - Yoda uses the DataCite metadata standard
> - DataverseNL uses the Dublin Core metadata standard
> - The VU Research Information System PURE uses the CERIF metadata standard

Reviewer: ```suggestion
Metadata standards allow for easier exchange of metadata and harvesting of the metadata by search engines. Many certified archives use a metadata standard for the descriptions. If you choose a data repository or registry, you should find out which metadata standard they use. At the VU the following standards are used:

- Yoda uses the DataCite metadata standard
- DataverseNL uses the Dublin Core metadata standard
- The VU Research Information System PURE uses the CERIF metadata standard
```

Adding whitespace.

Many archives implement or make use of specific metadata standards. The UK Digital Curation Centre (DCC) provides an [overview of metadata standards for different disciplines](https://www.dcc.ac.uk/guidance/standards/metadata). The list is a great and useful resource in establishing and carrying out your research methodology.

> ### Controlled Vocabularies & classifications

Reviewer: ```suggestion
### Ontologies
```

I know this as **Ontologies** – this section would be a candidate for its own topic instead of including in here, I propose.

Author: While I think that you may be correct, Chris, I'm a bit unsure about using ontologies as the heading, because it will be a word that many researchers don't know and that might scare them. I know the word scares me, because I don't know exactly what it means. What do others think?
